roles/nova-common/tasks/main.yml

```diff
@@ -4,7 +4,9 @@
         createhome=yes generate_ssh_key=yes ssh_key_bits=4096
 
 - name: nova packages
-  apt: pkg=python-numpy
+  apt: 
+    pkg="{{ item }}"
+  with_items: {{ nova.install_packages }}
```

Comment:
You shouldn't have to use the mustache brackets here, just reference the variable.